resources/rpi/usercfg.txt

```diff
@@ -0,0 +1,31 @@
+# Place "config.txt" changes (dtparam, dtoverlay, disable_overscan, etc.) in
+# this file. Please refer to the README file for a description of the various
+# configuration files on the boot partition.
+#
+# https://github.com/raspberrypi/firmware/blob/master/boot/overlays/README
+# PANOPTES POCS Pi customisation for Raspberry Pi 4B+
+
+dtoverlay=disable-bt
+dtoverlay=i2c-gpio
```

Comment:
Only the domepi needs GPIO, but it probably doesn't hurt to have it enabled on the others too.